reccmp/isledecomp/cvdump/symbols.py

```diff
@@ -105,7 +107,9 @@ def __init__(self):
         self.block_level: int = 0
 
     def read_line(self, line: str):
-        if (match := self._symbol_line_generic_regex.match(line)) is not None:
+        if len(line) == 0:
+            pass
```

Comment:
Likely a regression from #130 - I got tons of "Unhandled line:" messages on BETA10 with debug on